docker/supervisor-celery.conf

```diff
@@ -8,7 +8,7 @@ logfile=/code/logs/supervisord-celery.log
 # Changes:
 # - Set concurrency (number of workers) back to the default
 # - The code auto-reloads for development
-command=celery -A olympia.amo.celery:app worker --autoreload -E --loglevel=INFO
+command=watchmedo auto-restart -d src/ -p '*.py' -- celery -A olympia.amo.celery:app worker -E --loglevel=INFO
```

Comment:
what's this for?

Reply:
Celery 4 removed it's autoreload feature because it was buggy and unmaintained. watchdog replaces that for us now.